sdk/keyvault/azure-security-keyvault-common/vcpkg/CONTROL

```diff
@@ -1,3 +1,6 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# SPDX-License-Identifier: MIT
+#
 Source: azure-security-keyvault-common-cpp
```

Comment:
@antkmsft, @danieljurek will these comments be removed as part of the vcpkg publish step just like for the other packages like core?
https://github.com/Azure/azure-sdk-for-cpp/blob/46c84e3a3f1bc6c6f32c9892e94324a05009b232/sdk/core/azure-core/vcpkg/CONTROL#L1-L4

https://github.com/microsoft/vcpkg/blob/9d807041270262cc108b8deda60550a2cd95db1a/ports/azure-core-cpp/CONTROL#L1-L2

Reply:
@antkmsft should correct me if I'm wrong -- 

If the CMakeLists.txt file for the library is using `az_vcpkg_export` it should remove that section from the file: https://github.com/Azure/azure-sdk-for-cpp/blob/master/cmake-modules/AzureVcpkg.cmake#L64

This should also be verifiable in the package artifact outputs: https://dev.azure.com/azure-sdk/public/_build/results?buildId=830651&view=artifacts&pathAsName=false&type=publishedArtifacts 

If the comment removal script is working properly those lines will not be in the outputs.

Reply:
> If the CMakeLists.txt file for the library is using `az_vcpkg_export` 

Looks like it is:
https://github.com/Azure/azure-sdk-for-cpp/blob/308e5363adaa00ace2f06ff0f1a31747a0a28062/sdk/keyvault/azure-security-keyvault-keys/CMakeLists.txt#L105-L109

Verified in the artifacts, the script is working :)